v2/rscg_examples_site/docs/RSCG-Examples/GenerateDispose.md

```
@@ -163,7 +163,136 @@ Happy coding!
+
+
+[GenerateDispose.SourceGenerators.GenerateDispose(nameof(Drop))]
+
+
+partial class DALDB : IDisposable   // : IDisposable is optional!
+
+
+{
+
+
+    private ConnectionDB cn;
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)